packages/react/package.json

```diff
@@ -114,6 +114,7 @@
     "react-router-dom": "^6.2.1",
     "rimraf": "^3.0.2",
     "rollup": "^2.70.0",
+    "rollup-plugin-node-externals": "^4.1.1",
```

Comment:
Left a comment above that it would be nice not to have to add this dep.

Reply:
What do you mean by not adding? I am leaning towards make it explicit instead of having a phantom dep

Reply:
That's not the right way to go. You have to introduce a new one `@rollup/plugin-node-resolve` in order to use `/node_modules/` and that one will resolve the dependency imports to relative path in build artifacts (like `require('../../node_modules')`) which will break user stuff when you are not running in a mono repo

Reply:
One more thing I want to point out here is `/node_modules/` will cannot properly handle `tslib` which we do want to include in the bundle

Reply:
Makes sense, thanks for the explanation.